Gems/LmbrCentral/Code/Tests/BoxShapeTest.cpp

```diff
@@ -6,23 +6,24 @@
  *
  */
 
-#include <AzTest/AzTest.h>
-
+#include <AZTestShared/Math/MathTestHelpers.h>
 #include <AzCore/Component/ComponentApplication.h>
 #include <AzCore/Math/Matrix3x3.h>
 #include <AzCore/Math/Random.h>
-#include <AzFramework/Components/TransformComponent.h>
-#include <AzFramework/Components/NonUniformScaleComponent.h>
-#include <Shape/BoxShapeComponent.h>
 #include <AzCore/UnitTest/TestTypes.h>
-#include <AZTestShared/Math/MathTestHelpers.h>
+#include <AzFramework/Components/NonUniformScaleComponent.h>
+#include <AzFramework/Components/TransformComponent.h>
 #include <AzFramework/UnitTest/TestDebugDisplayRequests.h>
+#include <AzTest/AzTest.h>
+#include <Shape/BoxShapeComponent.h>
+#include <ShapeTestUtils.h>
 #include <ShapeThreadsafeTest.h>
 
 namespace UnitTest
 {
     class BoxShapeTest
         : public AllocatorsFixture
+        , public ShapeOffsetFixture
```

Comment:
question: Same question about Fixture (maybe it should just be renamed?)

Reply:
Renamed to ShapeOffsetTestsBase as suggested in your other comment.

Reply:
Perfect thank you!